AndroidPubSub/AuthenticatedPubsub.md

```diff
@@ -0,0 +1,85 @@
+## Using AWS IOT with user sign-in
+
+The sample demonstrates enabling guest users to use the AWS IoT APIs to securely publish-to and subscribe-from MQTT topics. Following steps demonstrates usage of AWS IoT with user sign-in. 
+
+1. Follow the steps [here](https://aws-amplify.github.io/docs/android/authentication) to add authentication UI to your app.
+
+2. Attach the following IAM Policy to the role of authenticated pool to authenticate and authorize the Cognito user to communicate with AWS IoT
```

Comment:
Talk about how you would do this - i.e., Use the AWS console, how to find the authenticated role, how to add the policy etc.